lib/fluent/plugin/out_mongo.rb

```diff
@@ -152,11 +150,19 @@ def emit(tag, es, chain)
     end
 
     def format(tag, time, record)
-      [time, record].to_msgpack
+      [tag, time, record].to_msgpack
```

Comment:
Using `[time, record].to_msgpack` and reuse `chunk.metadata.tag` to reduce serialization cost.